pilot/helpers/agents/Architect.py

```diff
@@ -68,6 +72,26 @@ def get_architecture(self):
         self.project.system_dependencies = llm_response["system_dependencies"]
         self.project.package_dependencies = llm_response["package_dependencies"]
 
+        warn_system_deps = [dep["name"] for dep in self.project.system_dependencies if dep["name"].lower() in WARN_SYSTEM_DEPS]
+        warn_package_deps = [dep["name"] for dep in self.project.package_dependencies if dep["name"].lower() in WARN_FRAMEWORKS]
+
+        if warn_system_deps:
+            print(color_yellow_bold(
+                f"Warning: GPT Pilot doesn't officially support {', '.join(warn_system_deps)}. "
+                f"You can try to use {'it' if len(warn_system_deps) == 1 else 'them'}, but you may run into problems."
+            ))
+            print('continue', type='buttons-only')
+            ask_user(self.project, "Press ENTER if you still want to proceed. If you'd like to modify the project description, close the app and start a new one.", require_some_input=False)
```

Comment:
Is the wording here clear? We want the users to either confirm or to close GPT Pilot and restart the whole process.